tsconfig.json

```diff
@@ -9,7 +9,17 @@
     "target": "es5",
     "typeRoots": ["./node_modules/@types"],
   },
-  "files": ["assets/constants.js", "assets/rich_text_components_definitions.js", "core/tests/karma-globals.ts", "core/tests/karma.conf.ts"],
-  "include": ["core", "extensions", "typings"],
-  "exclude": ["core/tests"]
+  "files": [
```

Comment:
Why are the new files included specifically and the core folder removed completely?

Reply:
Because  some are still included by `<script src="…"></script>` and so some files still need to be compiled separately.

Reply:
Ah OK, Thanks for the explanation!